.pre-commit-config.yaml

```diff
@@ -7,11 +7,11 @@ repos:
     hooks:
       - id: check-signoff
 
-  # Catch gofmt issues, if any.
-  - repo: git://github.com/dnephin/pre-commit-golang
-    rev: v0.3.5
+  - repo: https://github.com/golangci/golangci-lint
+    rev: v1.46.2
     hooks:
-      - id: go-fmt
+      - id: golangci-lint
+        entry: golangci-lint run --config=scripts/golangci.yml --fix --new
```

Comment:
`scripts/golangci.yml` is generated, you will need to run `make scripts/golangci.yml` first